src/enums/TransactionStatus.js

```diff
@@ -7,6 +7,15 @@ export const TransactionStatus = {
   ACCEPTED_ON_L1: 'ACCEPTED_ON_L1'
 };
 
+export const TransactionStatusFriendlyMessage = {
+  [TransactionStatus.REJECTED]: 'Invalid transaction',
+  [TransactionStatus.NOT_RECEIVED]: 'Waiting to be accepted on L2',
```

Comment:
In case of not received, I think we should specify it since it might indicate a problem and to distinguish between this and pending/received

Reply:
But the problem is that for example in the flow of transferring to StarkNet, we wait for 10 blocks to get a confirmation. In that time it will display not received instead of "waiting to be accepted on L2".
I think the former is better, wdyt?

Reply:
agree